examples/bri-3/src/bri/decorators/public-endpoint.ts

```diff
@@ -0,0 +1,4 @@
+import { SetMetadata } from '@nestjs/common';
```

Comment:
what is public endpoint?

Reply:
in order for login, or any, function to be invoked without any bearer token we need to have a way. This decorator does that job for us.

Reply:
@Manik-Jain so empty meta data signifies a login attempt?

Reply:
@Therecanbeonlyone1969 no, any end-points annotated with this decorator will not be picked by the Nest authenticator. 

From proposal doc, this decorator covers the proposed approach
`NestJs provides a capability to either manually update each end-point to run in an authenticated manner, or we can make all the end-points authenticated by default and choose which ones can be allowed public access. I propose to use the second approach`